platooning_control/src/platoon_control.cpp

```diff
@@ -109,17 +129,17 @@ namespace platoon_control
 
         return true;
     }   
-    
 
-
-    void  PlatoonControlPlugin::trajectoryPlan_cb(const cav_msgs::TrajectoryPlan::ConstPtr& tp){
-        
+    void  PlatoonControlPlugin::trajectoryPlan_cb(const cav_msgs::TrajectoryPlan::ConstPtr& tp)
+    {
         if (tp->trajectory_points.size() < 2) {
             ROS_WARN_STREAM("PlatoonControlPlugin cannot execute trajectory as only 1 point was provided");
             return;
         }
 
         latest_trajectory_ = *tp;
+        prev_input_time_ = ros::Time::now().toNsec() / 1000000;
```

Comment:
again Log statemet saying not received a new trajectory plan
Also, can you add log statements printing "prev_input_time_" and "tp->header.stamp.toNsec() / 1000000"
I think its better to use the trajectory plan's own timestamp, but I remember those didn't have reliable values in the past, so want to make sure.

Reply:
Sure.